lib/addons/karpenter/index.ts

```diff
@@ -13,7 +13,26 @@ import * as sqs from 'aws-cdk-lib/aws-sqs';
 import { Rule } from 'aws-cdk-lib/aws-events';
 import { SqsQueue } from 'aws-cdk-lib/aws-events-targets';
 import { Cluster } from 'aws-cdk-lib/aws-eks';
+import { EbsDeviceVolumeType } from 'aws-cdk-lib/aws-ec2';
 
+interface BlockDeviceMapping {
```

Comment:
These should be exported along with the `KarpenterAddOnProps` (not part of the PR atm) as customers will have issue consuming these otherwise.  Let's add `export` keyword to all public data types.

Reply:
thanks for the feedback. I've exported all the interfaces (including KarpenterAddOnProps) and now use these as part of the unit test.